src/Illuminate/Session/Store.php

```diff
@@ -449,11 +460,11 @@ public function decrement($key, $amount = 1)
     /**
      * Flash a key / value pair to the session.
      *
-     * @param  string  $key
+     * @param  string|\BackedEnum  $key
      * @param  mixed  $value
      * @return void
      */
-    public function flash(string $key, $value = true)
+    public function flash($key, $value = true)
```

Comment:
This is the one breaking change: removing the `string` type from the method.

I can revert this change and the doc if you like.